.github/workflows/silabs-open-csa-pr.yaml

```diff
@@ -0,0 +1,41 @@
+name: Open PR from csa to main
+
+permissions:
+    contents: write
+    pull-requests: write
+
+on:
+    workflow_run:
+        workflows: ["Daily Sync of the csa branch"]
+        types:
+            - completed
+    workflow_dispatch:
+
+jobs:
+    open-pr:
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  ref: main
+                  token: ${{secrets.WORKFLOW_TOKEN}}
+
+            - name: Checkout csa
+              run: |
+                  git fetch origin csa:csa
+                  git reset --hard csa
+
+            - name: Create Pull Request
+              uses: peter-evans/create-pull-request@v7
+              with:
+                  branch: automation/update_main
+                  base: main
+                  title: "Sync csa branch with main"
+                  body: |
+                      This PR syncs the csa branch with the main branch.
+
+                      **PR MUST BE MERGED WITH MERGE COMMIT - ADMIN MUST ENABLE THE OPTION**
+                  token: ${{secrets.GITHUB_TOKEN}}
+                  branch-token: ${{secrets.WORKFLOW_TOKEN}}
+                  labels: changing-submodules-on-purpose
```

Comment:
Can we add a label with maybe a special CI check that require admin merge for to ensure standard merge rather than squash merge

Reply:
added step to comment PR to remind merge commit